backend/app/api/api_v1/endpoints/bookmarks.py

```diff
@@ -0,0 +1,154 @@
+from fastapi import APIRouter, Depends, HTTPException
+from sqlalchemy.ext.asyncio import AsyncSession
+
+from app import crud, schemas
+from app.api import deps
+from app.core import auth
+
+router = APIRouter()
+
+
+@router.post("/create", response_model=schemas.BookmarkCreate)
+async def create_bookmark(
+    bookmark: schemas.BookmarkCreate, db: AsyncSession = Depends(deps.get_db)
+):
+    """
+    Create a new bookmark.
+
+    **Parameters**
+    * **obj_type**: type of object to bookmark
+    * **obj_id**: id of object to bookmark
+    * **user**: user id of user who created the bookmark
+
+    **Response**
+    * **id**: id of the bookmark
+    * **obj_type**: type of object to bookmark
+    * **obj_id**: id of object to bookmark
+    * **user**: user id of user who created the bookmark
+    """
+    return await crud.bookmark.create(db, obj_in=bookmark)
+
+
+@router.get("/list-all", response_model=list[schemas.BookmarkRead])
+async def list_bookmarks(skip: int = 0, limit: int = 100, db: AsyncSession = Depends(deps.get_db)):
+    """
+    List bookmarks. Available only for superusers.
+
+    **Parameters**
+    * **skip**: number of bookmarks to skip
+    * **limit**: maximum number of bookmarks to return
+
+    **Response**
+    * List of bookmarks
+    """
+    user = auth.fastapi_users.current_user(active=True, superuser=True)
+    if user:
+        return await crud.bookmark.get_multi(db, skip=skip, limit=limit)
+    else:
+        raise HTTPException(status_code=403, detail="Not enough permissions")
+
+
+@router.get("/get-by-id", response_model=schemas.BookmarkRead)
+async def get_bookmark(id: str, db: AsyncSession = Depends(deps.get_db)):
+    """
+    Get a bookmark by id. Available only for superusers.
+
+    **Parameters**
+    * **id**: id of the bookmark
```

Comment:
Is this the proper pydoc syntax? Can you include the API in sphinx documentation and check?